magento-research/pwa-studio
# [doc] Revise useEventListener doc blocks

## Description

Revise the doc blocks in the `useEventListener.js` file.

## Related Issue

Closes #1345 

## Verification Steps

1. Navigate to `pwa-devdocs` directory
2. Create the reference docs: `npm run build-reference-doc-snippets`
3. Verify a `useEventListener.md` file was created in `src/_includes/auto-generated/peregrine/src/hooks/` with doc block content from the `packages/peregrine/src/hooks/useEventListener.js` source file.

## Checklist:
